deploy/foreman-tasks.sysconfig

```diff
@@ -6,3 +6,15 @@ FOREMAN_LOGGING=warn
 FOREMAN_LOGGING_SQL=warn
 FOREMAN_TASK_PARAMS="-p foreman"
 FOREMAN_LOG_DIR=/var/log/foreman
+
+#Set the number of executors you want to run
+#EXECUTORS_COUNT=1
+
+#Set memory limit for executor process, before it's restarted automatically
+#EXECUTOR_MEMORY_LIMIT=1gb
+
+#Set delay before first memory polling to let executor initialize (in sec)
+#EXECUTOR_INIT_DELAY=180
+
+#Set memory polling interval, process memory will be checked every N seconds.
+#EXECUTOR_POLLING_INTERVAL=60
```

Comment:
@lzap The problem is that dynflow/foreman-tasks can't control how much memory a task consumes. 
I am not sure that the golden rule is applicable for background processing tasks such as package list or facts uploading. Those tasks consume a lot of memory (It's bloat symptoms and not leaks) for relatively short amount of time (around 10K objects each). Part of those objects could get stuck in memory (I can explain why in more details if you wish) and even for those that are not, the memory is not returned back to the OS. Our process of stopping the executor allows currently running jobs to finish before the shutdown, so I am not concerned about the same task being processed over and over again.
If you want the process to survive more time, you should increase its work memory threshold to a more reasonable value - that means one that allows processing of those huge tasks, if you think that the load is not excessive.
Restarting-in-a-loop would require multiple jobs with high memory consumption be scheduled one after the other. And even then the system will process them one by one, so the loop will end eventually.